src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,40 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/cars/drivers/addDriverToCar")
```

Comment:
check all endpoint names. Don't use camelCase, better use sub packages and one word for the name (if it's impossible, use a hyphen). pay attention to the link to your PR https://github.com/mate-academy/jv-web-practice/pull/411/files (without camelCase)
<img width="509" alt="image" src="https://user-images.githubusercontent.com/96740815/203334258-48fb9a32-a53a-4181-83c4-6e5d88439b33.png">